integration/git/new.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This was setup by performing `git init --bare` in a new folder and deleting all the .sample hooks that they add to that directory